FlipperKit.podspec

```diff
@@ -128,7 +129,8 @@ Pod::Spec.new do |spec|
   spec.subspec "FlipperKitLayoutComponentKitSupport" do |ss|
     ss.header_dir = "FlipperKitLayoutComponentKitSupport"
     ss.dependency             'FlipperKit/Core'
-    ss.dependency             'ComponentKit', '~> 0.0'
+    ss.dependency             'ComponentKit', '~> 0.30'
+    ss.dependency             'RenderCore', '~> 0.30'
```

Comment:
Current versions of Flipper rely on a header that exists in a different library than `ComponentKit`, namely `RenderCore`, and _only_ exists there starting from `v0.30`.

See:
* https://github.com/facebook/flipper/pull/1086/files#diff-364ae45351222c8099cb08ea04eacdeeR13
* https://github.com/facebook/flipper/commit/d0803ecd56c7e074de7ba3eb3af35ff3641553e2#diff-364ae45351222c8099cb08ea04eacdeeR13

Reply:
Note that I’m unsure how this has been building previously, as it seems that that change already exists since Flipper v0.38?